src/main/java/com/flightstats/hub/app/SingleHubBindings.java

```diff
@@ -27,6 +28,7 @@ protected void configure() {
         bind(ContentService.class).to(SingleContentService.class).asEagerSingleton();
         bind(DocumentationDao.class).to(FileDocumentationDao.class).asEagerSingleton();
         bind(ChannelTtlEnforcer.class).asEagerSingleton();
+        bind(JVMMetricsService.class).asEagerSingleton();
```

Comment:
Does this work if you move it from here (and the clustered hub bindings) and put it in plain old `HubBindings`?

Reply:
I am sure it will, for some reason I thought that was an interface the other two implemented and mostly ignored it

Reply:
Oh, maybe it is. I'll take a look.

Reply:
Yup. `HubBindings` is intended to be a place that all configurations of the Hub use. I would put this there as well.